src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -538,30 +556,54 @@ int32_t NaClSysOpen(struct NaClAppThread  *natp,
   char                 path[NACL_CONFIG_PATH_MAX];
   nacl_host_stat_t     stbuf;
   int                  allowed_flags;
+  const char           glibc_prefix[] = "/lib/glibc/";
```

Comment:
Is `char []` for a string preferred over `char *`?

Reply:
Ah yeah, that is a good idea, will fix.